src/components/dbManager.js

```diff
@@ -1,234 +1,304 @@
-const CouchDb = require('nano');
-import Utils from './utils';
+import Utils from './utils.js';
 import _ from 'lodash';
+import Db from "./db.js"
+import EncryptionUtils from "@verida/encryption-utils"
+import dotenv from 'dotenv';
+dotenv.config();
```

Comment:
I think you don't really need to have it in nearly every file, just server-app.js should be enough.